src/index.js

```diff
@@ -43,13 +43,28 @@ let rosNode = null;
 let pingMasterTimeout = null;
 
 //------------------------------------------------------------------
-
-function _checkMasterHelper(timeout=100) {
-  const localHelper = (resolve) => {
+/**
+ * @private
+ * Helper function to see if the master is available and able to accept
+ * connections.
+ * @param {number} timeout time in ms between connection attempts
+ * @param {number} maxTimeout maximum time in seconds to retry before timing out. 
```

Comment:
`rosnodejs` generally uses ms for time units - is there a reason you decided to go with seconds?